### PR TITLE
chore: update to Expo SDK 37.0.0

### PR DESCRIPTION
**Pull request by amandeepmittal**
Since the release of  latest Expo SDK, this PR update the Expo SDK to `37.x.x` and the related dependencies.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,26 +8,26 @@
...
-    "firebase": "6.6.0",
+    "expo": "^37.0.0",
+    "expo-asset": "~8.1.3",
+    "expo-font": "~8.1.0",
+    "firebase": "7.9.0",
     "formik": "2.1.4",
     "react": "16.9.0",
     "react-dom": "16.9.0",
-    "react-native": "https://github.com/expo/react-native/archive/sdk-36.0.1.tar.gz",
+    "react-native": "https://github.com/expo/react-native/archive/sdk-37.0.0.tar.gz",
```
Comment (byCedric) [inline package.json:18]: I think it's better to use `37.0.1` 😄 ([see this](https://github.com/expo/react-native/releases))

Comment (amandeepmittal) [reply package.json:18]: Ah, it slipped me. I didn't know about it. Also, creating a new expo project support the patch versions? I thought it only used latest major/minor released SDK versions.

**Comment by amandeepmittal** [comment]
I have upgraded the RN version to `37.0.1` as per your instructions @byCedric. Thank you so much for your time and reviewing these PRs. :)

**Comment by byCedric** [comment]
Anytime Aman ❤️ Thanks for making _and_ maintaining this!

**Pull request merged**